settings.gradle

```diff
@@ -1,8 +1,11 @@
 rootProject.name = 'dialogue'
 
+include 'dialogue-blocking-channels'
 include 'dialogue-client-test-lib'
 include 'dialogue-core'
 include 'dialogue-example'
+include 'dialogue-httpurlconnection-client'
+include 'dialogue-hc4-client'
 include 'dialogue-java-client'
```

Comment:
I know it's not this PR, but could we rename this to `dialogue-java9-client` as the `-java` suffix is a bit overloaded in conjure-land

Reply:
We can do that separately. I think email-service will need to be updated.